### PR TITLE
dbclient: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): v2022.83**

### CODE REVIEW

File: pages/linux/dbclient.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Connect to a remote host:
+
+`ssh {{user}}@{{host}}`
+
+- Connect to a remote host on [p]ort 2222:
+
+`ssh {{user}}@{{host}} -p 2222`
+
+- Connect to a remote host with using a specific [i]dentity key in dropbear format:
```
Comment (kbdharun) [inline pages/linux/dbclient.md:14]:
```suggestion
- Connect to a remote host using a specific [i]dentity key in dropbear format:
```

File: pages/linux/dbclient.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`ssh -i {{path/to/key}} {{user}}@{{host}}`
+
+- Run a command on the remote host with a [t]ty allocation allowing interaction with the remote command:
+
+`ssh {{user}}@{{host}} -t command arguments`
```
Comment (kbdharun) [inline pages/linux/dbclient.md:20]:
```suggestion
`ssh {{user}}@{{host}} -t {{command}} {{arguments}}`
```
How about this or

```suggestion
`ssh {{user}}@{{host}} -t {{command}} {{argument1 argument2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/dbclient.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# dbclient
+
+> Lightweight Dropbear Secure Shell client.
+> More information: <https://manned.org/dbclient.1>.
+
+- Connect to a remote host:
+
+`ssh {{user}}@{{host}}`
```
Comment (mfrw) [inline pages/linux/dbclient.md:8]: Should this be:
```suggestion
`dbclient {{user}}@{{host}}`
```

File: pages/linux/dbclient.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Connect to a remote host on [p]ort 2222:
+
+`ssh {{user}}@{{host}} -p 2222`
```
Comment (mfrw) [inline pages/linux/dbclient.md:12]:
```suggestion
`dbclient {{user}}@{{host}} -p 2222`
```

File: pages/linux/dbclient.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Connect to a remote host using a specific [i]dentity key in dropbear format:
+
+`ssh -i {{path/to/key}} {{user}}@{{host}}`
```
Comment (mfrw) [inline pages/linux/dbclient.md:16]:
```suggestion
`dbclient -i {{path/to/key}} {{user}}@{{host}}`
```

File: pages/linux/dbclient.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Run a command on the remote host with a [t]ty allocation allowing interaction with the remote command:
+
+`ssh {{user}}@{{host}} -t {{command}} {{argument1 argument2 ...}}`
```
Comment (mfrw) [inline pages/linux/dbclient.md:20]:
```suggestion
`dbclient {{user}}@{{host}} -t {{command}} {{argument1 argument2 ...}}`
```

File: pages/linux/dbclient.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Connect and forward [A]gent connections to remote host:
+
+`ssh -A {{user}}@{{host}}`
```
Comment (mfrw) [inline pages/linux/dbclient.md:24]:
```suggestion
`dbclient -A {{user}}@{{host}}`
```

File: pages/linux/dbclient.md (hunk 3 above)
Comment (kbdharun) [reply pages/linux/dbclient.md:8]: Oh, why didn't I notice it 😮

Comment (johandegn) [reply pages/linux/dbclient.md:8]: Oh, yes, of course. You are absolutely right. I am not sure how i managed to do that. I did test the examples with dbclient to ensure they worked as described in the manual, but it seems like i wrote them down too quickly. I was taking some inspiration from the ssh page, and might just have gotten too used to seeing them.

File: pages/linux/dbclient.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`dbclient {{user}}@{{host}}`
+
+- Connect to a remote host on [p]ort 2222:
+
+`dbclient {{user}}@{{host}} -p 2222`
+
+- Connect to a remote host using a specific [i]dentity key in dropbear format:
+
+`dbclient -i {{path/to/key}} {{user}}@{{host}}`
```
Comment (kbdharun) [inline pages/linux/dbclient.md:16]:
```suggestion
`dbclient -i {{path/to/key_file}} {{user}}@{{host}}`
```

File: pages/linux/dbclient.md (hunk 3 above)
Comment (mfrw) [reply pages/linux/dbclient.md]: No problemo folks. It happens :)
And this is what reviews are for :)
All cool. Thank you for the contribution :)

**Review by mfrw — APPROVED** [review]
LGTM :)

**Review by kbdharun — APPROVED** [review]

**Pull request merged**